src/shared/infrastructure/profile-metrics/profile-metrics.json

```diff
@@ -1,70 +1,70 @@
 {
   "block:contentions:count:contentions:count": {
     "id": "block:contentions:count:contentions:count",
-    "description": "Total count of blocking contentions",
+    "description": "Number of blocking contentions",
     "type": "contentions",
     "group": "block",
     "unit": "short"
   },
   "block:delay:nanoseconds:contentions:count": {
     "id": "block:delay:nanoseconds:contentions:count",
-    "description": "Total nanoseconds spent in blocking delays",
+    "description": "Time spent in blocking delays (nanoseconds)",
     "type": "delay",
     "group": "block",
     "unit": "ns"
   },
   "goroutine:goroutine:count:goroutine:count": {
     "id": "goroutine:goroutine:count:goroutine:count",
-    "description": "Current number of goroutines",
+    "description": "Number of goroutines",
     "type": "goroutine",
     "group": "goroutine",
     "unit": "short"
   },
   "goroutines:goroutine:count:goroutine:count": {
     "id": "goroutines:goroutine:count:goroutine:count",
-    "description": "Current number of goroutines",
+    "description": "Number of goroutines",
     "type": "goroutine",
     "group": "goroutine",
     "unit": "short"
   },
   "memory:alloc_in_new_tlab_bytes:bytes::": {
     "id": "memory:alloc_in_new_tlab_bytes:bytes::",
-    "description": "Total bytes allocated inside Thread-Local Allocation Buffers (TLAB)",
+    "description": "Size of memory allocated inside Thread-Local Allocation Buffers (bytes)",
```

Comment:
I personally find the TLAB bit useful. As that's how I know it and not "Thread-Local Allocation Buffers".